app/src/main/java/com/github/ppartisan/simplealarms/service/AlarmReceiver.java

```diff
@@ -93,7 +101,7 @@ public static void setReminderAlarm(Context context, Alarm alarm) {
         );
         final AlarmManager am = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
 
-        if(Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT) {
```

Comment:
If you're removing `Build.VERSION` from here because you added it to the static imports, wouldn't it also make sense to import `Build.VERSION_CODES.KITKAT` too, and remove `import android.os.Build;`?

Reply:
It would! It would be a little excessive to go back and change it now just for the sake of formatting, but you're right.

Reply:
Fair enough. Are there any benefits from doing imports that way (i.e. smaller file size or something)?

Reply:
Nope, I just usually find them easier to read (unless the class name(s) give some important context). It's purely subjective.